### DISCUSSION TITLE
mod_auth_openidc with Oracle IDCS ?

### DISCUSSION THREAD

**Discussion by PuneetDevops** [OpenIDC/mod_auth_openidc#1060] [General]
Hi Hans,

1) Can mod_auth_openidc be used with Oracle IDCS ?
2) I have used it Azure AD to enable SSO and aware of the process. Does it work the same way with IDCS as well or is there any concrete change ?
3) Is there any documentation around Oracle IDCS with mod_auth_openidc ?

Regards,
Puneet

**Comment by zandbelt** [comment]
Oracle IDCS is a certified OpenID Connect Provider so it can be used with mod_auth_openidc. I am not aware of any documentation on it, but being a regular Provider it should be straightforward as described in https://github.com/OpenIDC/mod_auth_openidc#quickstart-with-a-generic-openid-connect-provider

edit: google has https://blogs.oracle.com/analytics/post/oas-mp-sso-idcs-apache-openidc

**Reply by PuneetDevops** [reply]
Thanks Hans for this information. It Helps !!
